src/doc/src/reference/manifest.md

```diff
@@ -118,22 +118,21 @@ brackets at the end of each author.
 <a id="the-edition-field-optional"></a>
 #### The `edition` field
 
-You can opt in to a specific [Rust Edition] for your package with the
-`edition` key in `Cargo.toml`. If you don't specify the edition, it will
-default to 2015.
+The `edition` key is an optional key that affects which edition your package
+is compiled with. Cargo will always generate packages via [`cargo new`] with
+the `edition` key set to the latest edition. Setting the `edition` key in
```

Comment:
> Cargo will always generate packages via [`cargo new`] with
the `edition` key set to the latest edition.

Perhaps:
[`cargo new`] will generate a package with the `edition` key set to the latest edition.

Reply:
thanks, I like that wording much better. see my new changes!